src/support_sphere/lib/data/models/person.dart

```diff
@@ -0,0 +1,64 @@
+import 'package:equatable/equatable.dart';
+
+class Person extends Equatable {
```

Comment:
Can we add some documentation/comments about what a `Person` is? 

I'm sure it'll make sense once it's explained, but I'm not immediately having the easiest time distinguishing between a `Person`, a `Profile`, and a `User`